app/models/card.rb

```diff
@@ -55,7 +96,9 @@ def clear(field)
     field.squish.mb_chars.downcase.to_s unless field.nil?
   end
 
-  def set_default_review_date
-    self.review_date = Date.today + 3.days
+  def set_defaults
+    self.review_date = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.